c/kore/hello/conf/build.conf

```diff
@@ -0,0 +1,34 @@
+# hello build config
```

Comment:
`build.conf`?  Wouldn't CMake be far *FAR* more standard, shorter, and more readable, especially for a project like this?  Or what is this doing?  o.O?

Reply:
The `build.conf` is auto-generated by Kore's `kodev` command line tool and used for building project. Documentation is https://docs.kore.io/3.0.0/applications/building.html and https://docs.kore.io/3.0.0/applications/buildconf_reference.html .
I think this is the way of Kore. No other choice...

Reply:
Oh that looks horrifying...  Why would a project make their own build tool...  o.O

Reply:
just respect the tool design by kore